testing/runtests.py

```diff
@@ -391,14 +391,25 @@ def perform_test(self,testmgr):
 			elif (not outType and xopen(latex_output + "/temp",'r').read().find("Error")!= -1):
 				msg += ("PDF generation failed\n  For a description of the problem see 'refman.log' in the latex directory of this test",)
 				failed_latex=True
-			elif xopen(latex_output + "/refman.log",'r').read().find("Error")!= -1:
-				msg += ("PDF generation failed\n  For a description of the problem see 'refman.log' in the latex directory of this test",)
-				failed_latex=True
-			elif xopen(latex_output + "/refman.log",'r').read().find("Emergency stop")!= -1:
-				msg += ("PDF generation failed\n  For a description of the problem see 'refman.log' in the latex directory of this test",)
-				failed_latex=True
-			elif not self.args.keep:
-				shutil.rmtree(latex_output,ignore_errors=True)
+			else:
+				res = False;
+				try:
+					res = xopen(latex_output + "/refman.log",'r').read().find("Error")!= -1
+				except:
+					res = xopen(latex_output + "/refman.log",'r', encoding = "ISO-8859-1").read().find("Error")!= -1
```

Comment:
instead of doing this only as a fallback, I suggest to use this encoding (or "Latin-1", which is the same) from the start, which supposed to just read all characters as bytes and should always work. For just finding an ASCII string that should suffice I think.

Reply:
In this case it is indeed possible to use the `encoding='ISO-8859-1'` directly instead of the 
fallback solution.
I initially chose the fallback solution as not to endanger other tests.